code/arith.py

```diff
@@ -6,28 +7,35 @@ def main():
 
     operator = sys.argv[1]
     assert operator in ['add', 'subtract', 'multiply', 'divide'], (
-        'Operator is not one of add, subtract, multiply, or divide: '
-        'bailing out')
+        '''Operator is not one of add, subtract, multiply, or divide:
+        bailing out''')
     try:
         operand1, operand2 = float(sys.argv[2]), float(sys.argv[3])
     except ValueError:
         print('Cannot convert input to a number: bailing out')
-        return
+        return None
 
     do_arithmetic(operand1, operator, operand2)
+    return None
 
 
 def do_arithmetic(operand1, operator, operand2):
-
     if operator == 'add':
         value = operand1 + operand2
     elif operator == 'subtract':
         value = operand1 - operand2
     elif operator == 'multiply':
         value = operand1 * operand2
     elif operator == 'divide':
-        value = operand1 / operand2
+        try:
+            value = operand1 / operand2
+        except ZeroDivisionError:
+            print('Cannot divide by zero: bailing out')
+            return None
```

Comment:
Looking at this challenge in the lesson, it says "Write a command-line program that does addition and subtraction:" I am not sure why multiply and divide are included in the solution :/.

Reply:
That is probably a copy-paste issue, your suggestion is to remove multiplication and division?